### PR TITLE
CosmosDB Azure Function:  Adjustments for consistency 

### PR DESCRIPTION
**Pull request by AndyHale**
# CosmosDB Azure Function:  Adjustments for consistency 

## 📲 What

Adjustments to the Cosmos DB worker to make it more consistent with other projects in the overall solution: -

* Unit Tests assertion style matches other projects.
* Unit tests grouped into a Tests folder
* Newtonsoft.Json package used instead of System.Text.Jsoin to align with Service Bus library.

## 🤔 Why

After working on the Service Bus Worker I noticed some inconsistencies with the PR I made for this request, so these few adjustments make the fucntion project more consistent with the other projects.

## 🛠 How

Simple adjustmtments to test assertions and System.Text.Json replaced with Newtonsoft.Json.

## 👀 Evidence

The screenshot belwo shows the tests still pass after these adjutments,

image

### CODE REVIEW

**Review by sharonrussell — APPROVED** [review]

File: src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker.UnitTests/Tests/ChangeFeedListenerEventUnitTests.cs [hunk 1]
```diff
@@ -20,42 +20,30 @@ public void Constructor_ShouldAssignPropertiesCorrectly_WhenInitializedWithOpera
...
         // Assert
-        changeFeedEvent.Should().BeEquivalentTo(new
-        {
-            OperationCode = operationCode,
-            CorrelationId = correlationId,
-            EntityId = entityId,
-            ETag = eTag,
-            EventCode = (int)EventCode.EntityUpdated
-        });
+        changeFeedEvent.EventCode.Should().Be((int)EventCode.EntityUpdated);
```
Comment (alexanderwjrussell) [inline src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker.UnitTests/Tests/ChangeFeedListenerEventUnitTests.cs:23]: Much prefer this!

File: src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker.UnitTests/xxENSONOxx.xxSTACKSxx.Worker.UnitTests.csproj [hunk 2]
```diff
@@ -10,12 +10,12 @@
   <ItemGroup>
     <PackageReference Include="AutoFixture" Version="4.18.1" />
     <PackageReference Include="AutoFixture.AutoNSubstitute" Version="4.18.1" />
-    <PackageReference Include="FluentAssertions" Version="6.12.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageReference Include="FluentAssertions" Version="6.12.1" />
```
Comment (alexanderwjrussell) [inline src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker.UnitTests/xxENSONOxx.xxSTACKSxx.Worker.UnitTests.csproj:13]: Good shout bumping packages!

File: src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/CosmosDbChangeFeedEvent.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-using System.Text.Json.Serialization;
+using Newtonsoft.Json;
```
Comment (alexanderwjrussell) [inline src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/CosmosDbChangeFeedEvent.cs:1]: Where was this used that it was able to be swapped out without other code changes?

Comment (AndyHale) [reply src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/CosmosDbChangeFeedEvent.cs:1]: It was for the constructor attibute on line 7: -
*  [method: JsonConstructor]

I switched it for the Framework's version of the same name, but sicne realised the Service Bus library is using NewtonSoft, so thought I'd play it safe and be consistent.

**Pull request merged**